2021-02-04 19:47:50.069 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package azureml-train-automl missing from dependencies file.
2021-02-04 19:47:50.069 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: azureml-train-automl==1.20.0.*
2021-02-04 19:47:50.070 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package pandas missing from dependencies file.
2021-02-04 19:47:50.070 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: pandas==0.25.1
2021-02-04 19:47:50.071 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package psutil missing from dependencies file.
2021-02-04 19:47:50.071 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: psutil>5.0.0,<6.0.0
2021-02-04 19:47:50.072 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package scikit-learn missing from dependencies file.
2021-02-04 19:47:50.072 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: scikit-learn==0.22.1
2021-02-04 19:47:50.073 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package numpy missing from dependencies file.
2021-02-04 19:47:50.073 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: numpy~=1.18.0
2021-02-04 19:47:50.073 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package py-xgboost missing from dependencies file.
2021-02-04 19:47:50.074 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: py-xgboost<=0.90
2021-02-04 19:47:50.074 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package inference-schema missing from dependencies file.
2021-02-04 19:47:50.077 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package fbprophet missing from dependencies file.
2021-02-04 19:47:50.098 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: fbprophet==0.5
2021-02-04 19:47:50.099 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package setuptools-git missing from dependencies file.
2021-02-04 19:47:50.099 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:209 - Using installed version: setuptools-git==1.2
2021-02-04 19:47:50.161 - INFO - 5339 - azureml.train.automl._azureautomlclient._validate_input:664 - Start data validation.
2021-02-04 19:47:56.889 - CRITICAL - 5339 - azureml.train.automl._azureautomlclient.log_traceback:224 - Type: ExecutionFailure
Class: ValidationException
Message: ValidationException:
	Message: Failed to execute the requested operation: data/settings validation. Error details: Validation error(s): [{
    "additional_properties": {
        "debugInfo": null
    },
    "code": "UserError",
    "severity": 2,
    "message": "Y is empty",
    "message_format": "{dataName} is empty",
    "message_parameters": {
        "dataName": "Y"
    },
    "reference_code": null,
    "details_uri": null,
    "target": "training_data",
    "details": [
        {
            "additional_properties": {
                "debugInfo": null
            },
            "code": null,
            "severity": null,
            "message": "null",
            "message_format": null,
            "message_parameters": {},
            "reference_code": null,
            "details_uri": null,
            "target": null,
            "details": [],
            "inner_error": null
        }
    ],
    "inner_error": {
        "additional_properties": {},
        "code": "BadData",
        "inner_error": {
            "additional_properties": {},
            "code": "EmptyData",
            "inner_error": null
        }
    }
}, {
    "additional_properties": {
        "debugInfo": null
    },
    "code": "UserError",
    "severity": 2,
    "message": "YValid is empty",
    "message_format": "{dataName} is empty",
    "message_parameters": {
        "dataName": "YValid"
    },
    "reference_code": null,
    "details_uri": null,
    "target": "validation_size",
    "details": [
        {
            "additional_properties": {
                "debugInfo": null
            },
            "code": null,
            "severity": null,
            "message": "null",
            "message_format": null,
            "message_parameters": {},
            "reference_code": null,
            "details_uri": null,
            "target": null,
            "details": [],
            "inner_error": null
        }
    ],
    "inner_error": {
        "additional_properties": {},
        "code": "BadData",
        "inner_error": {
            "additional_properties": {},
            "code": "EmptyData",
            "inner_error": null
        }
    }
}]
	InnerException: None
	ErrorResponse 
{
    "error": {
        "code": "UserError",
        "message": "Failed to execute the requested operation: data/settings validation. Error details: Validation error(s): [{\n    \"additional_properties\": {\n        \"debugInfo\": null\n    },\n    \"code\": \"UserError\",\n    \"severity\": 2,\n    \"message\": \"Y is empty\",\n    \"message_format\": \"{dataName} is empty\",\n    \"message_parameters\": {\n        \"dataName\": \"Y\"\n    },\n    \"reference_code\": null,\n    \"details_uri\": null,\n    \"target\": \"training_data\",\n    \"details\": [\n        {\n            \"additional_properties\": {\n                \"debugInfo\": null\n            },\n            \"code\": null,\n            \"severity\": null,\n            \"message\": \"null\",\n            \"message_format\": null,\n            \"message_parameters\": {},\n            \"reference_code\": null,\n            \"details_uri\": null,\n            \"target\": null,\n            \"details\": [],\n            \"inner_error\": null\n        }\n    ],\n    \"inner_error\": {\n        \"additional_properties\": {},\n        \"code\": \"BadData\",\n        \"inner_error\": {\n            \"additional_properties\": {},\n            \"code\": \"EmptyData\",\n            \"inner_error\": null\n        }\n    }\n}, {\n    \"additional_properties\": {\n        \"debugInfo\": null\n    },\n    \"code\": \"UserError\",\n    \"severity\": 2,\n    \"message\": \"YValid is empty\",\n    \"message_format\": \"{dataName} is empty\",\n    \"message_parameters\": {\n        \"dataName\": \"YValid\"\n    },\n    \"reference_code\": null,\n    \"details_uri\": null,\n    \"target\": \"validation_size\",\n    \"details\": [\n        {\n            \"additional_properties\": {\n                \"debugInfo\": null\n            },\n            \"code\": null,\n            \"severity\": null,\n            \"message\": \"null\",\n            \"message_format\": null,\n            \"message_parameters\": {},\n            \"reference_code\": null,\n            \"details_uri\": null,\n            \"target\": null,\n            \"details\": [],\n            \"inner_error\": null\n        }\n    ],\n    \"inner_error\": {\n        \"additional_properties\": {},\n        \"code\": \"BadData\",\n        \"inner_error\": {\n            \"additional_properties\": {},\n            \"code\": \"EmptyData\",\n            \"inner_error\": null\n        }\n    }\n}]",
        "inner_error": {
            "code": "ExecutionFailure"
        }
    }
}
Traceback:
  File "_azureautomlclient.py", line 410, in fit
    test_data=test_data)
  File "_azureautomlclient.py", line 424, in _fit_remote
    validation_data=validation_data, test_data=test_data)
  File "_azureautomlclient.py", line 487, in _fit_remote_core
    training_data=training_data, validation_data=validation_data, test_data=test_data)
  File "_azureautomlclient.py", line 538, in _create_parent_run_for_remote
    test_data=test_data
  File "_azureautomlclient.py", line 617, in _create_and_validate_parent_run_dto
    self._validate_input(parent_run_dto)
  File "_azureautomlclient.py", line 694, in _validate_input
    ExecutionFailure, operation_name="data/settings validation", error_details=msg)

ExceptionTarget: Unspecified
2021-02-04 19:47:56.890 - INFO - 5339 - azureml.train.automl._azureautomlclient._fail_parent_run:847 - No parent run to fail
2021-02-04 19:48:18.708 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package azureml-train-automl missing from dependencies file.
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: azureml-train-automl==1.20.0.*
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package pandas missing from dependencies file.
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: pandas==0.25.1
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package psutil missing from dependencies file.
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: psutil>5.0.0,<6.0.0
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package scikit-learn missing from dependencies file.
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: scikit-learn==0.22.1
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package numpy missing from dependencies file.
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: numpy~=1.18.0
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package py-xgboost missing from dependencies file.
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: py-xgboost<=0.90
2021-02-04 19:48:18.709 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package inference-schema missing from dependencies file.
2021-02-04 19:48:18.710 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package fbprophet missing from dependencies file.
2021-02-04 19:48:18.710 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: fbprophet==0.5
2021-02-04 19:48:18.710 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package setuptools-git missing from dependencies file.
2021-02-04 19:48:18.710 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:209 - Using installed version: setuptools-git==1.2
2021-02-04 19:48:18.769 - INFO - 5339 - azureml.train.automl._azureautomlclient._validate_input:664 - Start data validation.
2021-02-04 19:48:27.036 - CRITICAL - 5339 - azureml.train.automl._azureautomlclient.log_traceback:224 - Type: ExecutionFailure
Class: ValidationException
Message: ValidationException:
	Message: Failed to execute the requested operation: data/settings validation. Error details: Validation error(s): [{
    "additional_properties": {
        "debugInfo": null
    },
    "code": "UserError",
    "severity": 2,
    "message": "Y is empty",
    "message_format": "{dataName} is empty",
    "message_parameters": {
        "dataName": "Y"
    },
    "reference_code": null,
    "details_uri": null,
    "target": "training_data",
    "details": [
        {
            "additional_properties": {
                "debugInfo": null
            },
            "code": null,
            "severity": null,
            "message": "null",
            "message_format": null,
            "message_parameters": {},
            "reference_code": null,
            "details_uri": null,
            "target": null,
            "details": [],
            "inner_error": null
        }
    ],
    "inner_error": {
        "additional_properties": {},
        "code": "BadData",
        "inner_error": {
            "additional_properties": {},
            "code": "EmptyData",
            "inner_error": null
        }
    }
}, {
    "additional_properties": {
        "debugInfo": null
    },
    "code": "UserError",
    "severity": 2,
    "message": "YValid is empty",
    "message_format": "{dataName} is empty",
    "message_parameters": {
        "dataName": "YValid"
    },
    "reference_code": null,
    "details_uri": null,
    "target": "validation_size",
    "details": [
        {
            "additional_properties": {
                "debugInfo": null
            },
            "code": null,
            "severity": null,
            "message": "null",
            "message_format": null,
            "message_parameters": {},
            "reference_code": null,
            "details_uri": null,
            "target": null,
            "details": [],
            "inner_error": null
        }
    ],
    "inner_error": {
        "additional_properties": {},
        "code": "BadData",
        "inner_error": {
            "additional_properties": {},
            "code": "EmptyData",
            "inner_error": null
        }
    }
}]
	InnerException: None
	ErrorResponse 
{
    "error": {
        "code": "UserError",
        "message": "Failed to execute the requested operation: data/settings validation. Error details: Validation error(s): [{\n    \"additional_properties\": {\n        \"debugInfo\": null\n    },\n    \"code\": \"UserError\",\n    \"severity\": 2,\n    \"message\": \"Y is empty\",\n    \"message_format\": \"{dataName} is empty\",\n    \"message_parameters\": {\n        \"dataName\": \"Y\"\n    },\n    \"reference_code\": null,\n    \"details_uri\": null,\n    \"target\": \"training_data\",\n    \"details\": [\n        {\n            \"additional_properties\": {\n                \"debugInfo\": null\n            },\n            \"code\": null,\n            \"severity\": null,\n            \"message\": \"null\",\n            \"message_format\": null,\n            \"message_parameters\": {},\n            \"reference_code\": null,\n            \"details_uri\": null,\n            \"target\": null,\n            \"details\": [],\n            \"inner_error\": null\n        }\n    ],\n    \"inner_error\": {\n        \"additional_properties\": {},\n        \"code\": \"BadData\",\n        \"inner_error\": {\n            \"additional_properties\": {},\n            \"code\": \"EmptyData\",\n            \"inner_error\": null\n        }\n    }\n}, {\n    \"additional_properties\": {\n        \"debugInfo\": null\n    },\n    \"code\": \"UserError\",\n    \"severity\": 2,\n    \"message\": \"YValid is empty\",\n    \"message_format\": \"{dataName} is empty\",\n    \"message_parameters\": {\n        \"dataName\": \"YValid\"\n    },\n    \"reference_code\": null,\n    \"details_uri\": null,\n    \"target\": \"validation_size\",\n    \"details\": [\n        {\n            \"additional_properties\": {\n                \"debugInfo\": null\n            },\n            \"code\": null,\n            \"severity\": null,\n            \"message\": \"null\",\n            \"message_format\": null,\n            \"message_parameters\": {},\n            \"reference_code\": null,\n            \"details_uri\": null,\n            \"target\": null,\n            \"details\": [],\n            \"inner_error\": null\n        }\n    ],\n    \"inner_error\": {\n        \"additional_properties\": {},\n        \"code\": \"BadData\",\n        \"inner_error\": {\n            \"additional_properties\": {},\n            \"code\": \"EmptyData\",\n            \"inner_error\": null\n        }\n    }\n}]",
        "inner_error": {
            "code": "ExecutionFailure"
        }
    }
}
Traceback:
  File "_azureautomlclient.py", line 410, in fit
    test_data=test_data)
  File "_azureautomlclient.py", line 424, in _fit_remote
    validation_data=validation_data, test_data=test_data)
  File "_azureautomlclient.py", line 487, in _fit_remote_core
    training_data=training_data, validation_data=validation_data, test_data=test_data)
  File "_azureautomlclient.py", line 538, in _create_parent_run_for_remote
    test_data=test_data
  File "_azureautomlclient.py", line 617, in _create_and_validate_parent_run_dto
    self._validate_input(parent_run_dto)
  File "_azureautomlclient.py", line 694, in _validate_input
    ExecutionFailure, operation_name="data/settings validation", error_details=msg)

ExceptionTarget: Unspecified
2021-02-04 19:48:27.036 - INFO - 5339 - azureml.train.automl._azureautomlclient._fail_parent_run:847 - No parent run to fail
2021-02-04 19:49:51.285 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package azureml-train-automl missing from dependencies file.
2021-02-04 19:49:51.285 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: azureml-train-automl==1.20.0.*
2021-02-04 19:49:51.285 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package pandas missing from dependencies file.
2021-02-04 19:49:51.285 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: pandas==0.25.1
2021-02-04 19:49:51.285 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package psutil missing from dependencies file.
2021-02-04 19:49:51.285 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: psutil>5.0.0,<6.0.0
2021-02-04 19:49:51.286 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package scikit-learn missing from dependencies file.
2021-02-04 19:49:51.286 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: scikit-learn==0.22.1
2021-02-04 19:49:51.286 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package numpy missing from dependencies file.
2021-02-04 19:49:51.286 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: numpy~=1.18.0
2021-02-04 19:49:51.286 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package py-xgboost missing from dependencies file.
2021-02-04 19:49:51.286 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: py-xgboost<=0.90
2021-02-04 19:49:51.286 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package inference-schema missing from dependencies file.
2021-02-04 19:49:51.286 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package fbprophet missing from dependencies file.
2021-02-04 19:49:51.286 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: fbprophet==0.5
2021-02-04 19:49:51.286 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package setuptools-git missing from dependencies file.
2021-02-04 19:49:51.286 - INFO - 5339 - azureml.train.automl._azureautomlclient.modify_run_configuration:209 - Using installed version: setuptools-git==1.2
2021-02-04 19:49:51.377 - INFO - 5339 - azureml.train.automl._azureautomlclient._validate_input:664 - Start data validation.
2021-02-04 19:49:56.233 - INFO - 5339 - azureml.train.automl._azureautomlclient._validate_input:677 - Validation service found the data has no errors.
2021-02-04 19:49:56.233 - INFO - 5339 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:542 - Start creating parent run.
2021-02-04 19:49:57.453 - INFO - 5339 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:562 - [ParentRunID:AutoML_685e55a8-5aa7-4709-8b5e-1091d140bf3c] Remote run using input X and y.
2021-02-04 19:49:57.933 - INFO - 5339 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:575 - Parent Run ID: AutoML_685e55a8-5aa7-4709-8b5e-1091d140bf3c
2021-02-04 19:49:57.934 - INFO - 5339 - azureml.train.automl._azureautomlclient._fit_remote_core:503 - Starting a snapshot run (snapshot_id : None)
2021-02-04 19:58:19.998 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package azureml-train-automl missing from dependencies file.
2021-02-04 19:58:19.998 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: azureml-train-automl==1.20.0.*
2021-02-04 19:58:19.998 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package pandas missing from dependencies file.
2021-02-04 19:58:19.998 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: pandas==0.25.1
2021-02-04 19:58:19.998 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package psutil missing from dependencies file.
2021-02-04 19:58:19.999 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: psutil>5.0.0,<6.0.0
2021-02-04 19:58:19.999 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package scikit-learn missing from dependencies file.
2021-02-04 19:58:19.999 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: scikit-learn==0.22.1
2021-02-04 19:58:19.999 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package numpy missing from dependencies file.
2021-02-04 19:58:19.999 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: numpy~=1.18.0
2021-02-04 19:58:19.999 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package py-xgboost missing from dependencies file.
2021-02-04 19:58:19.999 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: py-xgboost<=0.90
2021-02-04 19:58:19.999 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package inference-schema missing from dependencies file.
2021-02-04 19:58:20.000 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package fbprophet missing from dependencies file.
2021-02-04 19:58:20.000 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: fbprophet==0.5
2021-02-04 19:58:20.000 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package setuptools-git missing from dependencies file.
2021-02-04 19:58:20.000 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:209 - Using installed version: setuptools-git==1.2
2021-02-04 19:58:20.064 - INFO - 6084 - azureml.train.automl._azureautomlclient._validate_input:664 - Start data validation.
2021-02-04 19:58:25.512 - INFO - 6084 - azureml.train.automl._azureautomlclient._validate_input:677 - Validation service found the data has no errors.
2021-02-04 19:58:25.512 - INFO - 6084 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:542 - Start creating parent run.
2021-02-04 19:58:25.880 - INFO - 6084 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:562 - [ParentRunID:AutoML_18235cf5-7fb3-42ab-81e5-96747f2b0d3f] Remote run using input X and y.
2021-02-04 19:58:26.045 - INFO - 6084 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:575 - Parent Run ID: AutoML_18235cf5-7fb3-42ab-81e5-96747f2b0d3f
2021-02-04 19:58:26.046 - INFO - 6084 - azureml.train.automl._azureautomlclient._fit_remote_core:503 - Starting a snapshot run (snapshot_id : None)
2021-02-04 21:02:33.377 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package azureml-train-automl missing from dependencies file.
2021-02-04 21:02:33.377 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: azureml-train-automl==1.20.0.*
2021-02-04 21:02:33.378 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package pandas missing from dependencies file.
2021-02-04 21:02:33.378 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: pandas==0.25.1
2021-02-04 21:02:33.378 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package psutil missing from dependencies file.
2021-02-04 21:02:33.378 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: psutil>5.0.0,<6.0.0
2021-02-04 21:02:33.378 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package scikit-learn missing from dependencies file.
2021-02-04 21:02:33.378 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: scikit-learn==0.22.1
2021-02-04 21:02:33.379 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package numpy missing from dependencies file.
2021-02-04 21:02:33.379 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: numpy~=1.18.0
2021-02-04 21:02:33.379 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package py-xgboost missing from dependencies file.
2021-02-04 21:02:33.379 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: py-xgboost<=0.90
2021-02-04 21:02:33.379 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package inference-schema missing from dependencies file.
2021-02-04 21:02:33.379 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package fbprophet missing from dependencies file.
2021-02-04 21:02:33.379 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:205 - Using pinned version: fbprophet==0.5
2021-02-04 21:02:33.380 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:199 - Package setuptools-git missing from dependencies file.
2021-02-04 21:02:33.380 - INFO - 6084 - azureml.train.automl._azureautomlclient.modify_run_configuration:209 - Using installed version: setuptools-git==1.2
2021-02-04 21:02:33.445 - INFO - 6084 - azureml.train.automl._azureautomlclient._validate_input:664 - Start data validation.
2021-02-04 21:02:39.279 - INFO - 6084 - azureml.train.automl._azureautomlclient._validate_input:677 - Validation service found the data has no errors.
2021-02-04 21:02:39.280 - INFO - 6084 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:542 - Start creating parent run.
2021-02-04 21:02:40.436 - INFO - 6084 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:562 - [ParentRunID:AutoML_0f2d0a3d-f33f-490e-83d7-e6bbf8f85f59] Remote run using input X and y.
2021-02-04 21:02:40.946 - INFO - 6084 - azureml.train.automl._azureautomlclient._create_parent_run_for_remote:575 - Parent Run ID: AutoML_0f2d0a3d-f33f-490e-83d7-e6bbf8f85f59
2021-02-04 21:02:40.947 - INFO - 6084 - azureml.train.automl._azureautomlclient._fit_remote_core:503 - Starting a snapshot run (snapshot_id : None)
